app/templates/components/player-hud-components/player-infos.hbs

```diff
@@ -2,14 +2,15 @@
   <div class="img_container">
     <img src={{me.user.img_url}} />
   </div>
-  <div class="flex">
-    <div class="exp_level">Exp. level {{me.user.exp_level}}</div>
-  </div>
-  <div class="exp-meter">
-    <div class="exp-meter-progress"></div>
-    <div class="exp-meter-text">
-      {{me.user.experience}} / {{me.user.needed_exp_for_next_level}}
+
+  <div class="level-info-container__right">
+    <div class="exp_level">Exp. level
+      {{#lr-progress}}
+        template block text
+      {{/lr-progress}}
+      {{!-- {{me.user.exp_level}} --}}
     </div>
+    {{!-- {{lr-progress reached=me.user.experience total=me.user.needed_exp_for_next_level success=true}} --}}
```

Comment:
Yes... This was't ready as I tried to get the tests fixed. So now I'll make everything work as I normally would do and you fix the test later as you suggested :)